<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_stats</class>
 <widget class="QDialog" name="Dialog_stats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>621</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Image_Dark_PB">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>80</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Var(I-D) vs Mean(I-D)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="register_spinBox">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>180</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="Global_checkBox">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>210</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Global</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Image_PB">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>130</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Var(I) vs Mean(I-D)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title_Label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Slope_label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>320</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Slope = </string>
   </property>
  </widget>
  <widget class="QLabel" name="CorrelationCoeff_label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>360</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Correlation coefficient = </string>
   </property>
  </widget>
  <widget class="QPushButton" name="PTC_PB">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>240</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PTC Mean VS Var(σ²Shot)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PTC2_PB">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>270</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PTC Mean VS Var(σ²Read)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LnNdT_PB">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ln(Nd(T))</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Left_LineEdit">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>420</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Right_LineEdit">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>420</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Linear_PB">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>460</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear Regression</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
